src/parser/lval.js

```diff
@@ -378,4 +380,18 @@ export default class LValParser extends NodeUtils {
       }
     }
   }
+
+  checkToRestConversion(node: SpreadElement): void {
+    const validArgumentTypes = [
+      "Identifier",
+      "MemberExpression",
+      "AssignmentPattern",
```

Comment:
I just checked and it looks this was already a bug in babylon... so can you add a test? 😊